src/utils.js

```diff
@@ -85,6 +85,41 @@ exports.flattenStartCase = (object) => {
 };
 
 /**
+ * @summary Remove paths contained in strings
+ * @function
+ * @public
```

Comment:
Does this need to be public, or could it be private?
I think if it needs to be public, then it ought to have it's own set of unit-tests? (ideally)